go.mod

```diff
@@ -120,6 +120,7 @@ require (
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
 	github.com/klauspost/compress v1.16.0 // indirect
 	github.com/mailru/easyjson v0.7.6 // indirect
+	github.com/mattn/go-runewidth v0.0.13 // indirect
```

Comment:
You _could_ decide to bump this to v0.0.14 already; see https://github.com/docker/cli/pull/4251 (v0.0.14 looks to have various performance improvements)